src/utils/githubIssue.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs');
+const path = require('path');
+const { translateLanguage } = require('../languages/index');
+
+function getGitHubIssueURL(errorMessage) {
+  try {
+    const packageJsonPath = path.join(__dirname, '../../package.json');
+    const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, 'utf8'));
+
+    const issueBaseUrl = packageJson.bugs?.url;
```

Comment:
In this case since you're passing the title and the body, you would probably want to avoid the "Select a template" view

This code snippet should be able to remove the last `/choose` on the URL, leaving it as `https://github.com/TeamNovaSoft/novabot/issues/new`, which starts a new issue in a blank template

```suggestion
    const issueBaseUrl = packageJson.bugs?.url.replace(/\/choose$/, "");
```